src/nvim/option.c

```diff
@@ -2567,10 +2567,19 @@ did_set_string_option (
     // 'ambiwidth'
     if (check_opt_strings(p_ambw, p_ambw_values, false) != OK) {
       errmsg = e_invarg;
-    } else if (set_chars_option(&p_lcs) != NULL) {
-      errmsg = (char_u *)_("E834: Conflicts with value of 'listchars'");
-    } else if (set_chars_option(&p_fcs) != NULL) {
-      errmsg = (char_u *)_("E835: Conflicts with value of 'fillchars'");
+    } else {
+      FOR_ALL_TAB_WINDOWS(tp, wp) {
+        if (set_chars_option(wp, &wp->w_p_lcs) != NULL) {
+          errmsg = (char_u *)_("E834: Conflicts with value of 'listchars'");
+          goto ambw_end;
+        }
+        if (set_chars_option(wp, &wp->w_p_fcs) != NULL) {
+          errmsg = (char_u *)_("E835: Conflicts with value of 'fillchars'");
+          goto ambw_end;
+        }
+      }
+ambw_end:
+      {}  // clint prefers {} over ; as an empty statement
```

Comment:
Why not remove this line?  There's no need for a statement.

Reply:
Then I get this:

```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DINCLUDE_GENERATED_DECLARATIONS -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -D_GNU_SOURCE -Iconfig -I../src -Isrc/nvim/auto -Iinclude -I/usr/local/include/luajit-2.0 -isystem /usr/local/Cellar/libuv/1.25.0/include -isystem /usr/local/Cellar/msgpack/3.1.1/include -isystem /usr/local/include -isystem /usr/local/opt/gettext/include -Wshadow -Wconversion -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -g   -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -MD -MT src/nvim/CMakeFiles/nvim.dir/option.c.o -MF src/nvim/CMakeFiles/nvim.dir/option.c.o.d -o src/nvim/CMakeFiles/nvim.dir/option.c.o   -c ../src/nvim/option.c
../src/nvim/option.c:2582:5: error: expected statement
    }
    ^
1 error generated.
```

Reply:
I checked 6.8.1 from the [C99 draft](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf) and it seems as if a label _must_ be followed by a statement:

```
Syntax

    labeled-statement:
        identifier : statement
        [...]
```